packages/calcite-components/src/components/shell-panel/shell-panel.scss

```diff
@@ -288,16 +288,30 @@ slot[name="action-bar"]::slotted(calcite-action-bar),
   @apply border-color-3 border border-solid;
 }
 
-:host([position="start"]) slot[name="action-bar"]::slotted(calcite-action-bar),
-:host([position="start"]) .content ::slotted(calcite-flow),
-:host([position="start"]) .content ::slotted(calcite-panel) {
-  border-inline-start: none;
+:host([position="start"]:not([slot="panel-end"])),
```

Comment:
Can you clean up the duplicate selectors?

Reply:
these aren't duplicate selectors. The reason having them is to accommodate any existing code which do not specify `slot` for the `shell-panel` . Though it is expected to have shell-panel slotted inside shell , the current code does allow CSS to work without slot attribute. Make sense to drop support for such since it isn't technically a breaking change.

Reply:
nvm.